### PR TITLE
dts: msm8909: add acer-t01

### PR DESCRIPTION
**Pull request by wonderfulShrineMaidenOfParadise**
A fork of #155

### CODE REVIEW

**Comment by CE1CECL** [comment]
Hmm it doesn't seem to work with my catfish
Here is a summary of what did and didn't work:
1. https://github.com/wonderfulShrineMaidenOfParadise/lk2nd/commit/2d30efa2b02f2557e79fc69f823fcb62c36af1e7 : Instantly reboots and fastboot says "No device found"
2. https://github.com/TipzRickyCheung/lk2nd/commit/9fbf9b2d85bff29c1caf447a7024539ed12bc7b8 : No display, shows as Google fastboot but thats it (untested if commands work).
3. Qualcomm BSP from CodeAurora: While this was flashed to aboot, it stayed at a black screen but booted to OS just fine without display.
4. https://github.com/CE1CECL/lk2nd/commit/88c9b17643520e3e6d16e5adaa7a6c058ec0129a : Based off of S#1, same result, just adds stuff from S#2 and a DSI panel.

In other words, nothing worked for me except stock and S#3 was the closest I got in terms of building a QCOM bootloader.

**Comment by wonderfulShrineMaidenOfParadise** [comment]
@CE1CECL well that's expected. This series is not for catfish at all, and it's meant to add Acer Liquid Z330 (T01) support instead.
catfish patches are dropped since they are not relevant to my current goal, which is mainly for smartphones rather than feature phones/smartwatches.

**Comment by CE1CECL** [comment]
> @CE1CECL well that's expected. This series is not for catfish at all, and it's meant to add Acer Liquid Z330 (T01) support instead. catfish patches are dropped since they are not relevant to my current goal, which is mainly for smartphones rather than feature phones/smartwatches.

Found the issue was with your DTS, removing this line (https://github.com/CE1CECL/lk2nd/commit/a429756efb0bc7dd193066965771b037ed6f7793#diff-f9ca86b32911963355de13b1e049ccc44881732296c93761092db7864176e3e9L4) fixed it and I am still working on the display init'ing, however I keep getting this: 

```
root@HP-PAVILION-590:/r/lk/lk2nd# fastboot oem screenshot
                                                   FAILED (remote: 'display not initialized')
fastboot: error: Command failed
root@HP-PAVILION-590:/r/lk/lk2nd# 
```

Any Ideas? (My dts issue?)

**Comment by CE1CECL** [comment]
[stdout.txt](https://github.com/msm8916-mainline/lk2nd/files/11616757/stdout.txt)
lk_log

What does `[230] Invalid parameters for continuous splash` mean?

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Please joining  [#msm8916:postmarketos.org](https://app.element.io/#/room/#msm8916:postmarketos.org) for further discussions.
